From c931d83c8a460cb0ae1eb24f867e9f4e7194c911 Mon Sep 17 00:00:00 2001
From: Bruce Ashfield <bruce.ashfield@windriver.com>
Date: Tue, 3 Feb 2015 22:44:55 -0500
Subject: [PATCH 1/8] aufs: kbuild integration

upstream: git://git.code.sf.net/p/aufs/aufs3-standalone (aufs3-kbuild.patch)

Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 fs/Kconfig                | 1 +
 fs/Makefile               | 1 +
 include/uapi/linux/Kbuild | 1 +
 3 files changed, 3 insertions(+)

diff --git a/fs/Kconfig b/fs/Kconfig
index 6ccbee1c7dfe..b2427fd87aae 100644
--- a/fs/Kconfig
+++ b/fs/Kconfig
@@ -211,6 +211,7 @@ source "fs/ufs/Kconfig"
 source "fs/exofs/Kconfig"
 source "fs/f2fs/Kconfig"
 source "fs/efivarfs/Kconfig"
+source "fs/aufs/Kconfig"
 
 endif # MISC_FILESYSTEMS
 
diff --git a/fs/Makefile b/fs/Makefile
index 5c21042a2c56..8d0513ce092f 100644
--- a/fs/Makefile
+++ b/fs/Makefile
@@ -127,3 +127,4 @@ obj-$(CONFIG_CEPH_FS)		+= ceph/
 obj-$(CONFIG_PSTORE)		+= pstore/
 obj-$(CONFIG_EFIVAR_FS)		+= efivarfs/
 obj-$(CONFIG_YAFFS_FS)		+= yaffs2/
+obj-$(CONFIG_AUFS_FS)           += aufs/
diff --git a/include/uapi/linux/Kbuild b/include/uapi/linux/Kbuild
index 00b100023c47..e87bb1ed4feb 100644
--- a/include/uapi/linux/Kbuild
+++ b/include/uapi/linux/Kbuild
@@ -55,6 +55,7 @@ header-y += atmppp.h
 header-y += atmsap.h
 header-y += atmsvc.h
 header-y += atm_tcp.h
+header-y += aufs_type.h
 header-y += atm_zatm.h
 header-y += audit.h
 header-y += auto_fs4.h
-- 
2.1.0

